tests/help/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check commands help
+environment:
+  CMD/abort: abort
+  CMD/changes: changes
+  CMD/find: find
+  CMD/install: install
+  CMD/interfaces: interfaces
+  CMD/remove: remove
+execute: |
+  echo "Checking help for command $CMD"
+  expected="(?s)Usage:\n  snap \[OPTIONS\] $CMD.*?\n\nThe $CMD command .*?\nHelp Options:\n  -h, --help +Show this help message\n.*?"
+  actual=$(snap $CMD --help)
+  echo "$actual" | grep -Pzq "$expected" || exit 1
```

Comment:
Thanks @zyga, didn't know of expect before, looks nice and the specifications would be more readable. There are tests, like the login ones, which are a perfect fit for it. Let's see what the other members of the team think.